src/models/api-key.ts

```diff
@@ -39,39 +39,38 @@ const getApiKeysModel = function (
 		 * @description This method registers a new api key for the current user with the name given.
 		 *
 		 * @param {String} name - the API key name
+		 * @param {String} expiryDate - the API key expiry date
 		 * @param {String} [description=null] - the API key description
 		 *
 		 * @fulfil {String} - API key
 		 * @returns {Promise}
 		 *
 		 * @example
-		 * balena.models.apiKey.create(apiKeyName).then(function(apiKey) {
+		 * balena.models.apiKey.create(apiKeyName, 2030-10-12).then(function(apiKey) {
 		 * 	console.log(apiKey);
 		 * });
 		 *
 		 * @example
-		 * balena.models.apiKey.create(apiKeyName, apiKeyDescription).then(function(apiKey) {
+		 * balena.models.apiKey.create(apiKeyName, 2030-10-12, apiKeyDescription).then(function(apiKey) {
 		 * 	console.log(apiKey);
 		 * });
 		 */
 		async create(
 			name: string,
+			expiryDate: string | null,
```

Comment:
Let's make the api key methods accept an object since they got lengthy and re-ordering could lead to accidental user errors.

Then we might want to throw `if (!('expiryDate' in options))`.